common/changes/@microsoft/rush/CAPS-2946_2021-06-30-20-52.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "[rush] Prevent 'rush change' from prompting for an email address",
```

Comment:
```suggestion
      "comment": "[rush] Prevent \"rush change\" from prompting for an email address, since this feature is rarely used; to restore the old behavior, users can enable the \"includeEmailInChangeFile\" setting in version-policies.json",
```